src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
_WIN32 is the implementor name. The WIN32 is one defined that can be redefined by users. So the preferred way is _WIN32 to use the Microsoft definition.